libtransmission/torrent.cc

```diff
@@ -2478,7 +2463,10 @@ void tr_torrentsQueueMoveDown(tr_torrent* const* torrents_in, size_t torrent_cou
     std::sort(std::rbegin(torrents), std::rend(torrents), CompareTorrentByQueuePosition{});
     for (auto* tor : torrents)
```

Comment:
Note that my feeling is that this loop in `tr_torrentsQueueMoveDown` should be processed backwards, otherwise each sequence is swapping the position of the current torrent with the next one, so the next one is moved in the opposite direction as it's supposed to go.
This method is only used by transmission-**web**, so I can't verify its intended usage.